content-repo/extra-docs/releases/20.7.1.md

```diff
@@ -0,0 +1,360 @@
+# Cortex XSOAR Content Release Notes for version 20.7.1 (68719)
+##### Published on 21 July 2020
+
+---
+
+Welcome to the 20.7.1 Content release for Cortex XSOAR. Starting from the 20.6.0 release, we restructured our release notes to be based upon **Content Packs**. One of our team's top priorities is making our Content more accessible and understandable for both users and contributors. In this effort, we recently moved our [Content repo](https://github.com/demisto/content) to work in Packs format were there is a clear separation and grouping of Content artifacts. Each Content Pack provides a clear grouping of related Content artifacts used to either implement a use case, implement an integration or provide a clear set of functionality. Our new release notes are structured around Content Packs and you will see related Content artifacts grouped together according to Packs. We hope you will find this new format useful and clear.
```

Comment:
@kirbles19 we want to add to this disclaimer that we will still support installing content for earlier versions than 6.0
can you help please?

Reply:
What's the method for installing content in pre-6.0 deployments?

Reply:
same as today, just through the ui